styleguide.config.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
     var componentName = parts[parts.length - 1];
     return componentPath.replace(`/${componentName}`, `/docs/${componentName.replace('.js', '')}.md`);
   },
-  highlightTheme: 'pastel-on-dark',
```

Comment:
This config attribute is no longer supported by styleguidist.

Looks like we'll need to do more granular styling if we want to.